roguetown.dme

```diff
@@ -2832,6 +2832,7 @@
 #include "modular_causticcove\code\game\objects\items\clothes\stockings.dm"
 #include "modular_causticcove\code\game\objects\items\rogueitems\leash.dm"
 #include "modular_causticcove\code\game\objects\items\rogueitems\util.dm"
+#include "modular_causticcove\code\game\objects\items\weapons\ranged\ammo.dm"
```

Comment:
this is why you test shit lol

Even the simple changes

I completely missed this